Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM rocker/verse:4.0.4
-MAINTAINER support@civisanalytics.com
+ARG BUILDPLATFORM=linux/x86_64
+FROM --platform=${BUILDPLATFORM} rocker/verse:4.4.1
+LABEL org.opencontainers.image.authors="support@civisanalytics.com"
```

Comment:
`MAINTAINER` is [deprecated](https://docs.docker.com/reference/dockerfile/#maintainer-deprecated).